components/scream/src/interface/atm_comp_mct.F90.old

```diff
@@ -0,0 +1,119 @@
+module atm_comp_mct
```

Comment:
Is this file needed? Looks like it was a backup before you started doing mods. If so, let's remove it.